### PR TITLE
Restyle Gallery Page

### PR DESCRIPTION
**Pull request by jhancock532**
### [Link to GitPod Instance](https://gitpod.io/#https://github.com/jhancock532/bakerydemo/tree/restyle-gallery-page)

Adds container padding and grid layout for the gallery page, reuses the picture card styles while removing the link hover.

### Screenshots
<details><summary>Expand to view more</summary>

8080-jhancock532-bakerydemo-u3xd46cm0gj ws-eu44 gitpod io_gallery_(iPhone SE)

8080-jhancock532-bakerydemo-u3xd46cm0gj ws-eu44 gitpod io_gallery_ (2)

<a href="#">Back to top</a>
</details>

### Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Consider updating documentation. If you don't, tell us why.
- [x] If relevant, list the environments / browsers in which you tested your changes.

### CODE REVIEW

File: bakerydemo/static/css/main.css [hunk 1]
```diff
@@ -127,7 +127,8 @@ p {
   }
 }
 
-ol, ul {
+ol,
+ul {
```
Comment (jhancock532) [inline bakerydemo/static/css/main.css:131]: Auto-format on save

File: bakerydemo/static/css/main.css [hunk 2]
```diff
@@ -955,8 +951,47 @@ time.location-time {
...
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 10px;
+}
+
+@media (min-width: 766px) {
+  .gallery__container {
+    padding-top: 180px;
+    padding-bottom: 80px;
+
```
Comment (chris-lawton) [inline bakerydemo/static/css/main.css]:
```suggestion
```

File: bakerydemo/templates/tags/gallery.html [hunk 3]
```diff
@@ -1,12 +1,14 @@
 {% load wagtailimages_tags %}
 
 {% for img in images %}
-{% image img fill-285x200-c100 as img_obj %}
-<div class="col-sm-6">
-    <figure class="gallery-figure">
-        <img src="{{img_obj.url}}" class="img-responsive" />
-        <figcaption>{{ img.title }}</figcaption>
+{% image img fill-645x480-c100 as img_obj %}
```
Comment (chris-lawton) [inline bakerydemo/templates/tags/gallery.html]: question: can this be deleted now you have an image tag on line 7?

**Review by chris-lawton — APPROVED** [review]

File: bakerydemo/templates/tags/gallery.html (hunk 3 above)
Comment (jhancock532) [reply bakerydemo/templates/tags/gallery.html]: Good spot, thank you

**Pull request merged**